pkg/controller/elasticsearch/nodespec/podspec.go

```diff
@@ -245,3 +246,21 @@ func enableLog4JFormatMsgNoLookups(builder *defaults.PodTemplateBuilder) {
 		}
 	}
 }
+
+// Get contents of the script ConfigMap to generate config hash, excluding the suspended_pods.txt, as it may change over time.
+func getScriptsConfigMapContent(cm *corev1.ConfigMap) string {
```

Comment:
Maybe add a unit test for this function?

Reply:
Done